src/index.html

```diff
@@ -17,16 +17,36 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="search search--big"
+      data-qa="big"
+    >
+      <img
+        class="search__icon"
+        src="./images/Search.svg"
+        alt="search"
```

Comment:
The `alt` attribute for images should be unique and descriptive to improve accessibility. Since both images have the same `alt` text 'search', consider providing more context or differentiating them if they serve different purposes.